libs/core/src/lib/table/table.component.html

```diff
@@ -0,0 +1,63 @@
+<ng-template [fdSkeletonTemplate]="skeletonTemplate">
+    <ng-content></ng-content>
+</ng-template>
+
+<ng-template #skeletonTemplate>
```

Comment:
that's example how we add skeleton state for the complex components (multiple placeholders).